bin/bootstrax

```diff
@@ -624,7 +630,17 @@ def infer_mode(rd, input_dir):
         result = run_settings[eb]['very_high_rate']
     else:
         result = run_settings[eb]['max_rate']
-    log.info(f'Override processing mode for run {rd["number"]} changing to:\t {result}')
+    log.info(f'infer_mode::\tOverride processing mode for run {rd["number"]} changing to:'
+             f'\t {result}')
+
+    # Lower the settings if there are continuous failures on this run:
+    if rd['bootstrax'].get('n_failures', 0) >= lower_resources_after_n_failures:
+        result['cores'] //= 2
```

Comment:
Well you make a new ```run_settings``` every time overwriting any cashed ```run_settings``` so no this will not happen.
![afbeelding](https://user-images.githubusercontent.com/22295914/86479007-99473900-bd4b-11ea-9313-d4d785b544e4.png)

Reply:
If the memory model is what we'd normally expect, then there shouldn't be any issues

Reply:
To be absolutely sure I've changed it in 882c8b5. Additionally I changed some (old) code to also do this lowering of the resources if we end up with the default number of cores and messages (because the aggregate status isn't available or so)